### PR TITLE
More static services

### PR DESCRIPTION
**Pull request by imbstack**
This adds statsum and 90% of webhooktunnel/cloud-mirror. The hard part of webhooktunnel is how to get the certs in _or_ to now hack webhooktunnel such that it doesn't need certs. I think what I want to do right now is just deploy it as is and then have @jonasfj update the code later to rely on elb terminating ssl.

This also does not include any of the secrets for cloud-mirror-copier. I think I will ask @jhford to fill those in.

### CODE REVIEW

File: static-services.tf [hunk 1]
```diff
@@ -87,10 +106,69 @@ SECONDARY_SECRET=${var.stateless_dns_secondary_key}
...
+  image_hash               = "ed9a8c9f3949f3c8251d1d9a1ae3cfb0aaa99d584b707d12acfdf60f356cc3d5"
+  providers = {
+    aws = "aws.us-west-2"
+  }
+  env_vars = <<EOF
+PORT=12345
+ENV='production'
+HOSTNAME=${var.webhooktunnel_hostname}
+TASKCLUSTER_PROXY_SECRET_A=${var.webhooktunnel_secret_a}
+TASKCLUSTER_PROXY_SECRET_B=${var.webhooktunnel_secret_b}
```
Comment (djmitche) [inline static-services.tf:174]: That's a super confusing name :/

**Review by djmitche — APPROVED** [review]
I can't see anything to object to here. Does statsum really need such a beefy machine?

File: static-services.tf (hunk 1 above)
Comment (imbstack) [reply static-services.tf:174]: That's just what they chose :|

**Comment by imbstack** [comment]
I just used the same instance type that the one in docker cloud is now. We very well may be able to use a cheaper instance if we try! Adding jonasfj and jhford to the review now.

Note: This will take down stateless-dns for a few moments when we roll it out, but oh well!

File: static-services.tf [hunk 2]
```diff
@@ -87,10 +122,97 @@ SECONDARY_SECRET=${var.stateless_dns_secondary_key}
...
+TLS_KEY='HELP WHAT IS THE PLAN HERE'
+TLS_CERTIFICATE='HELP WHAT IS THE PLAN HERE'
```
Comment (jonasfj) [inline static-services.tf]: Is this static service created in k8s?

**Review by jhford — APPROVED** [review]
This looks reasonable to me, but I don't really know Terraform yet.  The values for the secrets can be obtained by running `docker-cloud stack inspect 66eb45ca` with these environment variables:

```
export DOCKERCLOUD_USER=...
export DOCKERCLOUD_PASS=...
export DOCKERCLOUD_NAMESPACE=moztc

**Comment by jonasfj** [comment]
@ckousik, you might be interested in what's going on here. Essentially, docker-cloud is shutting down with a quite short deadline, so we scrambling to migrated services off it. The plan is to run webhooktunnel behind a reverse proxy that terminates TLS (I'm working on a PR for this).

**Comment by ckousik** [comment]
@jonasfj  I saw the issue on the webhooktunnel repository. Will get to it tomorrow.

File: static-services.tf [hunk 3]
```diff
@@ -87,10 +122,97 @@ SECONDARY_SECRET=${var.stateless_dns_secondary_key}
...
+}
+
+output statsum_ips {
+  value = "${module.statsum.static_ips}"
 }
 
-output stateless_dns_eu_west_1_ip {
-  value = "${module.stateless_dns_eu_west_1.static_ip}"
+output webhooktunnel_ips {
+  value = "${module.webhooktunnel.static_ips}"
```
Comment (jonasfj) [inline static-services.tf:221]: What is this... maybe a doc comment? please...

Also what is it used for, I'm curious...

File: static-services.tf (hunk 2 above)
Comment (imbstack) [reply static-services.tf]: Nope, we gave up on that and are just spinning up static instances to serve this.

File: static-services.tf (hunk 3 above)
Comment (imbstack) [reply static-services.tf:221]: This is just the ip addr that we can point at to access the service.

**Pull request merged**